docs/content/docs/deep/summaryTelemetry.md

```diff
@@ -252,40 +239,72 @@ The event cancels in response to a summary nack op for this attempt, an error al
 
 - `message` - message indicating result of summarize attempt; possible values:
 
+  - `disconnect` - the summary op was submitted but broadcast was cancelled.
   - `submitSummaryFailure` - the attempt failed to submit the summary op.
   - `summaryOpWaitTimeout` - timeout while waiting to receive the submitted summary op broadcasted.
   - `summaryAckWaitTimeout` - timeout while waiting to receive a summary ack/nack op in response to this attempt's summary op.
   - `summaryNack` - attempt was rejected by server via a summary nack op.
   - `summaryAck` - attempt was successful, and the summary ack op was received.
 
-- `timeWaiting` (ack/nack received only) - time in ms spent waiting for the summary ack/nack op after submitting the summary op.
-- `sequenceNumber` (ack/nack received only) - sequence number of the summary ack/nack op in response to this attempt's summary op.
+- `ackWaitDuration` (ack/nack received only) - time in ms spent waiting for the summary ack/nack op after submitting the summary op.
+- `ackNackSequenceNumber` (ack/nack received only) - sequence number of the summary ack/nack op in response to this attempt's summary op.
 - `summarySequenceNumber` (ack/nack received only) - sequence number of this attempt's summary op.
 - `handle` (ack only) - summary handle found on this attempt's summary ack op.
 
-### GenerateSummary
+### generate
```

Comment:
is this the name of the event? Shouldn't it be upper case?

Reply:
I agree. I don't know when it became lowercase, but I'll change it.

Reply:
I take it back. This isn't the name of the event. It's a suffix. I'll update the md file.